test/quantization/pt2e/test_xnnpack_quantizer.py

```diff
@@ -582,6 +582,63 @@ def test_dynamic_linear_int4_weight(self):
                 qconfig_mapping,
             )
 
+    def test_weight_only_linear_int4_weight(self):
+        quantizer = XNNPACKQuantizer()
+        quantization_config = get_symmetric_quantization_config(
+            is_per_channel=True,
+            is_weight_only=True,
+            weight_qmin=0,
+            weight_qmax=15,
```

Comment:
```suggestion
            weight_qmin=-8,
            weight_qmax=7,
```

Reply:
I thought xnnpack is using 0, 15? that's why they have zero_point = 8 right?

Reply:
This feels like better UX with our symmetric quant theme for xnnpack. We can shift the weights to zp=8 and make then uint8 during the lowering.